CHANGELOG.md

```diff
@@ -1,290 +1,268 @@
+- [BUGFIX] Fix reading SDK attributes in Objective-C. See [#654][]
+- [FEATURE] RUM: Track slow UI renders with RUM Long Tasks. See [#567][]
+- [FEATURE] RUM: Add API to notify RUM session start: `.onRUMSessionStart(_: (String, Bool) -> Void)`. See [#590][]
+- [FEATURE] Logs: Add logs scrubbing API: `.setLogEventMapper(_: (LogEvent) -> LogEvent)`. See [#640][]
+- [FEATURE] Add `Datadog.isInitialized` API. See [#566][]
+- [FEATURE] Add API for clearing out all SDK data: `Datadog.clearAllData()`. See [#644][]
+- [FEATURE] Add support for `us5` site. See [#576][]
+- [FEATURE] Support `URLSession` proxy configuration with `.connectionProxyDictionary`. See [#582][]
+- [IMPROVEMENT] Compress HTTP body in SDK uploads. See [#626][]
+- [IMPROVEMENT] Change type of `.xhr` RUM Resources to `.native`. See [#605][]
+- [IMPROVEMENT] Link logs and traces to RUM Actions. See [#615][]
+- [IMPROVEMENT] Crash Reporting: Fix symbolication issue for iOS Simulator crashes. See [#563][]
+- [IMPROVEMENT] Fix various typos in docs. See [#569][] (Thanks [@michalsrutek][])
+- [IMPROVEMENT] Use Intake API V2 for SDK data uploads. See [#562][]
 
 # 1.7.2 / 11-8-2021
 
 ### Changes
 
-* [BUGFIX] Fix iOS 15 crash related to `ProcessInfo.isLowPowerModeEnabled`. See [#609][] [#655][] (Thanks [@pingd][])
+- [BUGFIX] Fix iOS 15 crash related to `ProcessInfo.isLowPowerModeEnabled`. See [#609][] [#655][] (Thanks [@pingd][])
 
 # 1.7.1 / 10-4-2021
 
 ### Changes
 
-* [BUGFIX] Fix iOS 15 crash in `MobileDevice.swift`. See [#609][] [#613][] (Thanks [@arnauddorgans][], [@earltedly][])
-* [BUGFIX] RUM: Fix bug with "Refresh Rate" Mobile Vital reporting very low values. [#608][]
+- [BUGFIX] Fix iOS 15 crash in `MobileDevice.swift`. See [#609][] [#613][] (Thanks [@arnauddorgans][], [@earltedly][])
+- [BUGFIX] RUM: Fix bug with "Refresh Rate" Mobile Vital reporting very low values. [#608][]
 
 # 1.7.0 / 09-27-2021
 
 ### Changes
 
-* [BUGFIX] RUM: Fix `DDRUMView` API visibility for Objective-C. See [#583][]
-* [FEATURE] Crash Reporting: Add `DatadogCrashReporting`
-* [FEATURE] RUM: Add Mobile Vitals. See [#493][] [#514][] [#522][] [#495][]
-* [FEATURE] RUM: Add option for renaming instrumented actions. See [#539][]
-* [FEATURE] RUM: Add option for tracking events when app is in background. See [#504][] [#537][]
-* [FEATURE] Add support for `us3` site. See [#523][]
-* [IMPROVEMENT] RUM: Improve RUM <> APM integration. See [#524][] [#575][] [#531][] (Thanks [@jracollins][], [@marcusway][])
-* [IMPROVEMENT] RUM: Improve naming for views started with `key:`. See [#534][]
-* [IMPROVEMENT] RUM: Improve actions instrumentation. See [#509][] [#545][] [#547][]
-* [IMPROVEMENT] RUM: Sanitize custom timings for views. See [#525][]
-* [IMPROVEMENT] Do not retry uploading events if Client Token is invalid. See [#535][]
+- [BUGFIX] RUM: Fix `DDRUMView` API visibility for Objective-C. See [#583][]
+- [FEATURE] Crash Reporting: Add `DatadogCrashReporting`
```

Comment:
Any reason for changing `*` into `-` in the entire changelog? I'm a bit afraid this diff can cause problems - although in theory should not.

Reply:
Yea it should be ok! @louiszawadzki any reason for changing this? #982

Reply:
@ncreated It has already been merge into `develop`, so it shouldn't conflict in future merges 👍